modules/Sources/UIComponents/TextFields/TCATextField/TCATextFieldStore.swift

```diff
@@ -16,9 +16,14 @@ public struct TCATextFieldReducer: Reducer {
         public var text = "".redacted
         public var valid = false
 
-        public init(validationType: String.ValidationType?, text: RedactableString) {
+        public init(
+            validationType: String.ValidationType?,
+            text: RedactableString,
+            valid: Bool = true
```

Comment:
I don't understand why the default initialization of this field (which I presume is the cached "field is valid" state from the UI) is being changed to `true`, or why that is happening here instead of line 17.

Reply:
Good catch, I needed to expose `valid` variable for tests and by default it must be false. Fixed